imap/message.c

```diff
@@ -1051,10 +1061,21 @@ int imap_fetch_message(struct Context *ctx, struct Message *msg, int msgno)
    * command handler */
   h->active = false;
 
-  snprintf(buf, sizeof(buf), "UID FETCH %u %s", HEADER_DATA(h)->uid,
-           (mutt_bit_isset(idata->capabilities, IMAP4REV1) ?
-                (ImapPeek ? "BODY.PEEK[]" : "BODY[]") :
-                "RFC822"));
+  /* the IMAP4REV1 predicate being true will always result in one of our two
```

Comment:
Sorry, but I find the original code clearer.

Reply:
Figured this might be a bit too weird; will revert this as well.